content/posts/newsletter/2020-11-05.md

```diff
@@ -0,0 +1,174 @@
+url: 2020/10/newsletter-10-october/
+date: 2020-11-05 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of October 2020.
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- French on [linuxfr.org](https://linuxfr.org/tags/xmpp/public) and [jabberfr.org](https://news/jabberfr.org/category/newsletter/)
+
+## XSF Announcements
+
+The XSF started the application period for its annual board & council elections. Applicants for the council must be elected XSF members, while board seats are open to everyone. If you are interested in applying for any of these positions please visit the application page [here](https://wiki.xmpp.org/web/Board_and_Council_Elections_2020).
+
+The Q4 2020 XSF membership application period is open as well. When you are interested to become an XSF member you can apply [here](https://wiki.xmpp.org/web/Membership_Applications_Q4_2020) before 8th November.
+
+## Articles
+
+MongooseIM takes advantage of XMPP's extensibility to allow its users to customise their solution based on their needs.
+A great example of this is their [case study with Pando Health](https://www.erlang-solutions.com/resources/case-studies.html#replacing-whatsapp-and-legacy-pagers-with-mongooseim-in-healthcare-29-doc), an award-winning chat system for health care that is now the most used offering in the United Kingdom National Health Service (NHS).
+
+The DBJR (German Federal Youth Council) tries to educate their member organisations about open source and open standards. In our digital tools' blog, Jonah from the Kaidan project wrote an article for lesser-techie people to explain why they develop Kaidan XMPP client. [The article is in German.](https://tooldoku.dbjr.de/2020/10/kaidan-auf-dem-weg-zur-einfachen-sicheren-und-dezentralen-chat-app/) (DE)
+
+When [Snikket](https://snikket.org/) launched back at FOSDEM 2020 in February its website was rather basic, so the team just added a blog section and the very first two posts expanding on the [introduction](https://snikket.org/blog/introducing-snikket/) and announcing the [Snikket CIC](https://snikket.org/blog/snikket-cic/), a legal company that backs Snikket. The post has the gritty legal details, but one thing stands out, it now allows Snikket to get funded via donations and grants. In good tradition there's also a [blog RSS feed](https://snikket.org/blog/index.xml) so you can stay up to date.
+
+The IETF (yes, the Internet Engineering Task Force) deployed a trial instance of an XMPP service for their operational experience with local registration, guest account access, and provides a web client. These services are available at [xmpp-trial1.ietf.org](https://xmpp-trial1.ietf.org/). Great thanks to the XSF members supporting to make this happen! You can read more about the background and configuration of the deployment on the [Prosody blog](https://blog.prosody.im/xmpp-at-the-ietf/).
```

Comment:
@mwild1 The Prosody blog post is **not** from October

Reply:
I can change the date on the post if that helps? :)

Seriously, does it matter? The activity that it discusses happened in October. Why should we wait another 1+ months before we share it with people? Do we want to be in the habit of only sharing old news? That's not what other communities do, and it's not what people want or expect.

By the time the next newsletter is published, the IETF trial will have been running for two months already, and it will be one month away from being shut down.

It's not the first time I've had people push against including stuff because it is the "wrong month". It's a rule I would like to see go, for the sake of improving the freshness of our content.

Reply:
> By the time the next newsletter is published, the IETF trial will have been running for two months already, and it will be one month away from being shut down.

Actually it would be just a few weeks before the end of the trial, because the November newsletter won't be published until a week into December... :man_shrugging:

Reply:
I understand, we had this discussion in the past, and was somewhat agreed that we stick with only those from that month. Eg. Why isn't Snikket 2.9.0 announced too, then?

@Echolon Thoughts?

Reply:
Explicitly limiting content of a newsletter to old news makes little sense to me. News, by definition, is ... new.

Reply:
Also news never stops, so you can't ever send the newsletter since you need to add one more thing. The cut off date, last day of month, should ensure an editorial control of that. This is published on the 5-6 day of the next month to give reviewers time to read and contribute to the existing news, not to cram some more links.

This was discussed in the MUC, please restart this discussion there.

Reply:
I agree thats a bit sad that we may cut off news even we technically can possibly involve them.

However, I prefer to follow what @licaon-kter says, not only for redactional reasons, but also for ressources. We cannot handle more releases currently. We also cannot compete against feed aggregators. So lets keep it in general like this for a more wider community.

Still, and in this case, I am happy to make exceptions. I think that was important work and the "event" already ends by december. So from my side is fine to keep it.

Everyone happy?

Reply:
Happy for now :) Can continue in the MUC.